Downloading (…)lve/main/config.json: 100%|██████████████████████████████████████████████████████████████████| 694/694 [00:00<00:00, 217kB/s]
Downloading (…)lve/main/config.json:   0%|                                                                        | 0.00/694 [00:00<?, ?B/s]

























Some weights of the model checkpoint at allenai/longformer-base-4096 were not used when initializing LongformerForTokenClassification: ['lm_head.dense.bias', 'lm_head.layer_norm.weight', 'lm_head.dense.weight', 'lm_head.layer_norm.bias', 'lm_head.decoder.weight', 'lm_head.bias']
- This IS expected if you are initializing LongformerForTokenClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing LongformerForTokenClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of LongformerForTokenClassification were not initialized from the model checkpoint at allenai/longformer-base-4096 and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Traceback (most recent call last):
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/train.py", line 21, in <module>
    main('fbp2_LongFormer_config.json', CFG)
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/train.py", line 16, in main
    getattr(train_loop, cfg.loop)(cfg)
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/trainer/train_loop.py", line 31, in train_loop
    model, criterion, val_metrics, optimizer, lr_scheduler = train_input.model_setting(len(train))
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/trainer/trainer.py", line 246, in model_setting
    model = getattr(model_arch, self.cfg.model_arch)(self.cfg)
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/model/model.py", line 134, in __init__
    reinit_topk(self.model, cfg.num_reinit)
  File "/home/qcqced/바탕화면/ML_Test/FeedBackPrize_2/model/model_utils.py", line 88, in reinit_topk
    model.encoder.layer[-num_layers:].apply(model._init_weights)
  File "/home/qcqced/anaconda3/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1614, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'LongformerForTokenClassification' object has no attribute 'encoder'